index.html

```diff
@@ -4372,6 +4372,154 @@ <h2>Revision History</h2>
         </li>
       </ul>
     </section>
+    <section class="appendix informative">
+      <h2>Understanding Proof Sets and Proof Chains</h2>
+      <p>
+Proof sets and chains allow multiple proofs to be carried in a VC, i.e., instead
+of a single <em>proof</em> included in the signed VC we can express multiple
+proofs in an array as shown in the previous examples. The elements of this array
+are members of <em>proof set</em> and, optionally, a <em>proof chain</em>.
+The purpose of this section is to explain the intended use of each of these entities
+and, in particular, their differing security properties. These differing security
+properties lead to differences in the processing in section <a href="#add-proof-set-chain"></a>.
+      </p>
+      <p>
+We are going to represent VCs and their proofs in a highly abbreviated manner so
+that the key security properties can be observed.
+      </p>
+      <p>
+We consider a scenario with three signatories: a CEO, a CFO, and a VP of
+Engineering. Each will need to have a public and private key pair for signing a
+document. We denote the private/public keys of each of these signatories by
+<em>priv/pubCEO</em>, <em>priv/pubCFO</em>, and <em>priv/pubVPE</em>
+respectively.
+      </p>
+      <p>
+When constructing a <em>proof set</em> where each of the signatories signs a
+document without concern we construct a proof symbolically as:
+      </p>
+      <pre><code class="language-javascript">
+{
+  &quot;type&quot;: &quot;DataIntegrityProof&quot;,
+  &quot;cryptosuite&quot;: &quot;jcs-eddsa-2022&quot;,
+  &quot;created&quot;: &quot;2023-03-05T19:23:24Z&quot;,
+  &quot;proofPurpose&quot;: &quot;assertionMethod&quot;,
+  &quot;verificationMethod&quot;: pubCEO,
+  &quot;proofValue&quot;: signature(privCEO, document)
+}
+      </code></pre>
+      <p>
+Where <em>pubCEO</em> is used as a placeholder for a reference that resolves to
+the CEO's public key and <code>signature(priv, doc)</code> denotes the
```

Comment:
I get the usage of public key and private key here, but it might be confusing to readers that have seen us use "verification method" and "verification material" throughout the document. Public/private key cryptography is not the only type of proof that can be attached to VCs, so we have to be careful when we use that terminology when what we really mean is "any digitally expressible mathematical proof"... such as proof of work, proof of stake, biometric template verification, and a variety of other "non PKI-based proofs".